src/NServiceBus.AmazonSQS/Configure/RequiresWindows.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NServiceBus
+{
+    using System.Runtime.InteropServices;
+    using System.Threading.Tasks;
+    using Transport;
+
+    class RequiresWindows
+    {
+        public Task<StartupCheckResult> Validate()
+        {
+            var isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
While using `RuntimeInformation.IsOSPlatform` is the better, preferred API for several reasons, including that it can distinguish between linux and macOS, it might be simpler to use `Environment.OSVersion.Platform == PlatformID.Win32NT` instead. This would avoid the need for the `System.Runtime.InteropServices.RuntimeInformation` package dependency.

Reply:
Is there a significant downside to having the dependency on `System.Runtime.InteropServices.RuntimeInformation`?

Reply:
Given that the need for `RuntimeInformation.IsOSPlatform` would be temporary, we'd be adding a dependency that is also temporary. You could argue that removing a dependency would be a breaking change, especially given how transitive NuGet dependencies work these days, so we'd either have to keep the dependency around even though we don't need it, or release a new major just to let linux work once the bug is fixed.

Reply:
Good argument @bording and nice insights. I suggest we switch to @bording suggestion?